addons/scopes/stringtable.xml

```diff
@@ -365,5 +365,25 @@
             <Chinesesimp>此模块可为高倍率瞄准镜新增归零风偏，距离用的调整纽。</Chinesesimp>
             <Chinese>此模塊可為高倍率瞄準鏡新增歸零風偏，距離用的調整紐。</Chinese>
         </Key>
+        <Key ID="STR_ACE_Scopes_DisplayAdjustmentUp">
```

Comment:
There seems something wrong with the translations.
https://github.com/acemod/ACE3/pull/5702#issuecomment-340248564